backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -403,18 +389,11 @@ export class ContributionResolver {
       withDeleted: true,
       relations: ['user'],
     })
-    if (!emailContact) {
-      logger.error(`Could not find UserContact with email: ${email}`)
-      throw new Error(`Could not find UserContact with email: ${email}`)
+    if (!emailContact || !emailContact.user) {
+      throw new LogError('Could not find User', email)
     }
-    const user = emailContact.user
```

Comment:
Why remove this after check if emailContact.deletedAt we never need emailContact but the user in it!

Reply:
this way is more memory efficient. else we have two copies of the user object in memory.